modules/ROOT/pages/db/db-connector-index.adoc

```diff
@@ -21,15 +36,53 @@ The Database Connector provides specialized support for the following database:
 
 You can set up other JDBC databases using a generic JDBC configuration. You can also reference a JDBC `DataSource` object or a `XADataSource` object, which is required for XA transactions. You typically create the object in Studio using Spring.
 
+== Database Listeners and Operations
+
+The Database connector provides a listener to read from a database in the Source section of a flow.
+You can execute other operations which also allow you to read and write to a database anywhere in the Process section.
+For example, instead of writing single records to a database, bulk operations allow you to modify a collection of records by simply supplying the SQL to modify each record.
+Other operations allow you to carry out Data Definition Language (DDL) operations, to execute stored procedures, or to execute entire SQL scripts at once.
+
+== Audience
+
+* Starting user -- Read this Get Started document and create your Mule app in
+xref:database/database-connector-design-center.adoc[Design Center] or
+xref:database/database-connector-studio.adoc[Anypoint Studio].
+The xref:database/database-connector-config-topics.adoc[Additional Configuration Information] topic
```

Comment:
The Additional Configuration Information link does not display correctly. Needs to be fixed.